issues.js

```diff
@@ -168,23 +189,32 @@ function groupIssuesByTopic(issues) {
 // Append the give issue to the appropriate Topic list
 function renderIssue(issue) {
     var rendered_issue = Handlebars.templates.issues(issue);
-    var $topic_group = $(sanitizeId(issue.topic));
+    var $topic_group = $(sanitizeId(issue.category) + " " + sanitizeId(issue.topic));
     $topic_group.removeClass("hide");
     $topic_group.children("ul").append(rendered_issue);
 }
 
 // Render the containers for each Topic list
-function renderTopics(topics) {
-    var topic_obj = {};
-    topic_obj.topics = topics;
-    
-    var rendered_topics = Handlebars.templates.topic_groups(topic_obj);
-    $("#issues_list").html(rendered_topics);
+function renderGroupings(topics, categories) {
+    var rendered_topics = Handlebars.templates.topic_groups({
+      topics: topics
+    }); // render a topics snippet to be used by each category
+
+    var rendered_categories = Handlebars.templates.categories({
+      categories: categories
+    }); // render the categories
+
+    $("#null_category").html(rendered_topics); // a bucket for all issues without a category
+
+    $("#issues_list").append(rendered_categories);
+    $.each(categories, function (key, category) {
+      $(sanitizeId(category)).append(rendered_topics);
```

Comment:
In the current implementation I had gone with using classes for the elements inside of "Programming" or "User Feedback", but the solution you proposed could be an alternative.